boards.txt

```diff
@@ -63,6 +63,18 @@ Nucleo_144.menu.pnum.NUCLEO_F767ZI.build.product_line=STM32F767xx
 Nucleo_144.menu.pnum.NUCLEO_F767ZI.build.variant=NUCLEO_F767ZI
 Nucleo_144.menu.pnum.NUCLEO_F767ZI.build.cmsis_lib_gcc=arm_cortexM7l_math
 
+# NUCLEO_H743ZI board
+Nucleo_144.menu.pnum.NUCLEO_H743ZI=Nucleo H743ZI
+Nucleo_144.menu.pnum.NUCLEO_H743ZI.node=NODE_H743ZI
+Nucleo_144.menu.pnum.NUCLEO_H743ZI.upload.maximum_size=2097152
+Nucleo_144.menu.pnum.NUCLEO_H743ZI.upload.maximum_data_size=131072
```

Comment:
Isn't the data size a little bit too small? Doesn't the H743ZI have 1 MiB of RAM?

Reply:
Yes, but as only DTCMRAM is used in linker file I've put it to 128k.
https://github.com/stm32duino/Arduino_Core_STM32/pull/392/files#diff-d5d8843f8b8eee9af248bdc07c943d85R63

Reply:
Oh I see. The H7 memory layout is pretty different than the F7 memory layout. Good to know. The DTCM wasn't at the start location on the F7, big gotcha for the H7. But okay, good to know thx.